server.php

```diff
@@ -65,9 +65,11 @@ function valet_support_xip_io($domain)
 $valetSitePath = null;
 
 foreach ($valetConfig['paths'] as $path) {
-    if (is_dir($path.'/'.$siteName)) {
-        $valetSitePath = $path.'/'.$siteName;
-
+    $domain = ($pos = strrpos($siteName, '.')) !== false
+                ? substr($siteName, $pos+1)
+                : null;
+    if (is_dir($path.'/'.$siteName) || is_dir($path.'/'.$domain)) {
```

Comment:
Sorry, could not get this to work, could not get wildcard sub-domains to work since 2 days now,, tried on homestead, valet,,, but :( just isn;t moving, requesting for help pls ...